<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>El_GUI_CLIENTO</class>
 <widget class="QMainWindow" name="El_GUI_CLIENTO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>el_gui_cliento v.0.0.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>600</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <widget class="QWidget" name="Main_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QFrame" name="left_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>580</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>580</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QGroupBox" name="main_groupB">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>370</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>370</height>
           </size>
          </property>
          <property name="title">
           <string>el_gui_cliento</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QGroupBox" name="time_groupB">
             <property name="minimumSize">
              <size>
               <width>170</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>90</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="HH">
                <item>
                 <widget class="QLCDNumber" name="HH_lcdN">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>75</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>75</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::Panel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <property name="smallDecimalPoint">
                   <bool>false</bool>
                  </property>
                  <property name="digitCount">
                   <number>2</number>
                  </property>
                  <property name="mode">
                   <enum>QLCDNumber::Mode::Dec</enum>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::SegmentStyle::Filled</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="MM">
                <item>
                 <widget class="QLCDNumber" name="MM_lcdN">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>75</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>75</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::Panel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Shadow::Raised</enum>
                  </property>
                  <property name="smallDecimalPoint">
                   <bool>false</bool>
                  </property>
                  <property name="digitCount">
                   <number>2</number>
                  </property>
                  <property name="mode">
                   <enum>QLCDNumber::Mode::Dec</enum>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::SegmentStyle::Filled</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="plugin_layout_1">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLineEdit" name="plugin_led_lineE_1">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="plugin_toolB_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>plugin_name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="plugin_layout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLineEdit" name="plugin_led_lineE_2">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="plugin_toolB_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>plugin_name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="plugin_layout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLineEdit" name="plugin_led_lineE_3">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="plugin_toolB_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>plugin_name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="plugin_layout_4">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLineEdit" name="plugin_led_lineE_4">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="plugin_toolB_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>plugin_name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="plugin_layout_5">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLineEdit" name="plugin_led_lineE_5">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="plugin_toolB_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>plugin_name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QGroupBox" name="options_groupB">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="title">
           <string>Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <property name="spacing">
            <number>5</number>
           </property>
           <item row="1" column="0">
            <widget class="QLineEdit" name="network_stat_line">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>170</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>35</height>
              </size>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QToolButton" name="settings_toolB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>170</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="right_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>580</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
